tests/unittest/disaggregated/test_router.py

```
@@ -854,6 +855,97 @@ async def test_kv_cache_aware_router_polls_kv_cache_events(
+
+    req0 = CompletionRequest(model="TinyLlama", prompt=[prefix])
+    server0, info0 = await router.get_next_server(req0)
+    await router.finish_request(req0)
+
+    expected_blocks = {h for hl in info0["block_hashes"] for h in hl}
+    assert expected_blocks
+    assert expected_blocks.issubset(
+        router._server_state[server0]._kv_cache_block_table)
+    assert router._server_state[server0]._kv_events_applied == 0
+
+    req1 = CompletionRequest(model="TinyLlama", prompt=[prefix + [42]])
```

Ruff (0.15.18)

[warning] Consider `[*prefix, 42]` instead of concatenation

Replace with `[*prefix, 42]`

(RUF005)